src/VisualStudio/CSharp/Impl/LanguageClient/CSharpLanguageServerClient.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Remote;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.LanguageServices.Implementation.LanguageService;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.LanguageClient
+{
+    // currently, platform doesn't allow multiple content types
+    // to be associated with 1 ILanguageClient forcing us to
+    // create multiple ILanguageClients for each content type
+    [ContentType(ContentTypeNames.CSharpContentType)]
+    [Export(typeof(ILanguageClient))]
+    [ExportMetadata("Capabilities", "WorkspaceStreamingSymbolProvider")]
+    internal class CSharpLanguageServerClient : AbstractLanguageServerClient
+    {
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public CSharpLanguageServerClient(
+            VisualStudioWorkspace workspace,
+            LanguageServerClientEventListener eventListener,
+            IAsynchronousOperationListenerProvider listenerProvider)
+            : base(workspace,
+                eventListener,
+                listenerProvider,
+                WellKnownServiceHubServices.CSharpLanguageServer,
+                "ManagedLanguage.IDE.CSharpLanguageServer")
+        {
+        }
+
+        public override string Name => CSharpVSResources.CSharp_language_server_client;
```

Comment:
Where is this string visible to the user? Just wondering if the current terminology will be easy to understand for the users..

Reply:
Why is this localized?

Reply:
following contract
https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.languageserver.client.ilanguageclient.name?view=visualstudiosdk-2019#Microsoft_VisualStudio_LanguageServer_Client_ILanguageClient_Name

I dont actually know where it is displayed. tagging @tinaschrepfer